### PR TITLE
Fix button spacing and add-to-wishlist on community app pages

### PR DESCRIPTION
**Pull request by candela97**

### CODE REVIEW

**Review by MxtOUT — COMMENTED** [review]
Not a fan of those template literals starting at the end of the line.
And what was fixed here?

**Comment by candela97** [comment]
> Not a fan of those template literals starting at the end of the line.
> And what was fixed here?

Restores a space between buttons, if there isn't one already:
螢幕擷取畫面 2020-12-18 204244

File: src/js/Content/Features/Community/App/FCommunityAppPageWishlist.js [hunk 1]
```diff
@@ -61,12 +50,11 @@ export default class FCommunityAppPageWishlist extends Feature {
             const action = wishlisted ? "wishlist.remove" : "wishlist.add";
 
             try {
-                await Background.action(action, that.appid);
+                await Background.action(action, appid);
 
                 successBtn.style.display = wishlisted ? "none" : "";
                 addBtn.style.display = wishlisted ? "" : "none";
 
-                DynamicStore.clear();
```
Comment (candela97) [inline src/js/Content/Features/Community/App/FCommunityAppPageWishlist.js:69]: This is already called in Background.action ?

Comment (MxtOUT) [reply src/js/Content/Features/Community/App/FCommunityAppPageWishlist.js:69]: You're right

**Comment by MxtOUT** [comment]
I see now, I'll add a comment describing the necessity of those white spaces

**Pull request merged**